src/simplify_algebra.cpp

```diff
@@ -482,17 +482,40 @@ struct find_double_add_lit_broadcast
     }
 };
 
+/// Find elementswise operators that have all broadcast inputs. It then
+/// rewrites the elementwise to do the computation on the non-broadcasted
+/// axes, and then broadcast that result.
```

Comment:
Could you add (not necessarily on this line) a definition of an inner broadcast and an inner broadcast axis.  It's not self-evident.

Reply:
`find_inner_broadcast` is the name of the matcher/finder(and this comment explains what that is looking for). Without a larger context, I dont know what an inner broadcast axis is, and I cant find anywhere this term is used.

Reply:
I'm not going to explain what it means, because I'm ***asking*** what it means.

Reply:
I dont know what inner broadcast axis is either, and I dont even know why its relevant to this PR.